frontend/src/services/auth.js

```diff
@@ -55,6 +56,36 @@ export async function loginUser(email, password) {
     data: data,
   };
 
+  try {
+    const response = await axios.request(config);
+
+    if (response.status === 200) {
+      return response.data.user;
+    } else {
+      return null;
+    }
+  } catch (error) {
+    return null;
+  }
+}
+
+export async function registerStudentId(id, ID_No) {
+  const data = JSON.stringify({
+    token: id,
+    ID_No: ID_No,
+  });
+
+  let config = {
+    method: "post",
+    maxBodyLength: Infinity,
+    url: "http://localhost:8000/auth/google/register",
```

Comment:
```suggestion
    url: `${process.env.BACKEND_URL}/auth/google/register`,
```

Reply:
To keep testing & development easier, I suggest configuring these later

Reply:
Sure, if you want you can add them later as well.
Just so you know, adding them in env is going to make testing and development even easier, suppose you want to change the base url in the API calls in the frontend, then in the current scenario you'll have to find all the endpoints and then change it everywhere, but if you set it in the env file, you change it at one place and everything will work fine.